### DISCUSSION TITLE
求助，/api/fs/list refresh=True，用python request发包，没有刷新是为什么？

### DISCUSSION THREAD

**Discussion by ashesofdream** [AlistGo/alist#6805] [Q&A]
pydebug:
data
{'path': '/pikpak_webdav/My Pack', 'password': '', 'page': 1, 'per_page': 0, 'refresh': True}

url
'http://192.168.xxx.xxx:5244/api/fs/list'

headers
{'Authorization': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFkbWluIiwicHdkX3RzIjoxNzE4MjEz...jB11bBueGXnjMUNJL0EYAzFrI1z_OxUifgaGmL1E9Q', 'Referer': 'http://192.168.194.10:5244/pikpak_webdav/My%20Pack', 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36 Edg/126.0.0.0', 'Origin': 'http://192.168.xxx.xxx:5244/pikpak_webdav/My%20Pack'}

rp =requests.post(url,headers=headers,data=data)

实测，点击网页的刷新按钮是确实能刷新的，但我用python发包就不行，求助

**Comment by dr34m-cn** [comment]
data=data改为json=data

**Comment by ashesofdream** [comment]
不知道为什么，更新+重启后又好了，估计触发什么奇怪bug了.....
